tests/utils.test.ts

```diff
@@ -198,3 +200,168 @@ Deno.test({
     );
   },
 });
+
+Deno.test({
+  name: "[collection] Create a collection",
+  fn: async (t) => {
+    const collection = new Collection();
+
+    assertExists(collection);
+
+    await t.step({
+      name: "[collection] collection values to array",
+      fn: async (t) => {
+        const testCollection = new Collection([["best", "tri"], ["proficient", "yui"]]);
+
+        assertEquals(testCollection.array(), ["tri", "yui"]);
+      },
+    });
+
+    await t.step({
+      name: "[collection] get a random value",
+      fn: async (t) => {
+        const testCollection = new Collection([["best", "tri"]]);
+
+        assertEquals(testCollection.random(), "tri");
+        assertEquals(collection.random(), undefined);
+      },
+    });
```

Comment:
@TriForMine Is this the equivalence for `const true = Math.random() < 0.5, false = Math.random() < 0.5`

Reply:
In other words, these two lines
```
const testCollection = new Collection([["best", "tri"]]);

        assertEquals(testCollection.random(), "tri");
```
Looks like there is a 50/50 chance it will fail/pass, right?

Reply:
Oh lol, that's not what I wanted to do